src/components/HistoryItem/HistoryItem.tsx

```diff
@@ -0,0 +1,38 @@
+import { Link } from 'react-router-dom'
+import cn from 'classnames'
+
+import { Button } from '@/components'
+import { useTheme } from '@/context'
+import { useAppDispatch } from '@/store'
+import { removeHistoryItem } from '@/store/historySlice'
+
+import s from './HistoryItem.module.css'
+
+interface HistoryItemProps {
+  query: string
+}
+
+export const HistoryItem = ({ query }: HistoryItemProps) => {
+  const dispatch = useAppDispatch()
+  const { isTheme } = useTheme()
```

Comment:
```suggestion
  const { theme } = useTheme()
```
уместнее, так как не булево значение